stdoutcallback/results/ansiblePlaybookJSONResults.go

```diff
@@ -161,6 +161,8 @@ func skipLine(line string) bool {
 	skipPatterns := []string{
 		// This pattern skips timer's callback whitelist output
 		"^[\\s\\t]*Playbook run took [0-9]+ days, [0-9]+ hours, [0-9]+ minutes, [0-9]+ seconds$",
+		"^\\[WARNING\\]:.+ use",
```

Comment:
Hi @apenella

Yes sure `^\\[WARNING\\]:` is fine are you keeping the `^\\-vvvv` ?

I still had problems, when matching  only for `^\\[WARNING\\]:.+ use`, since where separate lines, is it related to terminal size ?

stdoutcallback/results/ansiblePlaybookJSONResults.go
```golang
for scanner.Scan() {
		line := scanner.Text()
                fmt.Printf("%t => %s", skipLine(line), line)
		if !skipLine(line) {
			fmt.Fprintf(w, "%s", line)
		}
	}
```

Output
```bash
true =>     "stats": {
true =>         "lb-01": {
true =>             "changed": 0,
true =>             "failures": 0,
true =>             "ignored": 0,
true =>             "ok": 1,
true =>             "rescued": 0,
true =>             "skipped": 0,
true =>             "unreachable": 0
true =>         },
true =>         "lb-02": {
true =>             "changed": 0,
true =>             "failures": 0,
true =>             "ignored": 0,
true =>             "ok": 1,
true =>             "rescued": 0,
true =>             "skipped": 0,
true =>             "unreachable": 0
true =>         },
true =>         "master-01": {
true =>             "changed": 0,
true =>             "failures": 1,
true =>             "ignored": 0,
true =>             "ok": 2,
true =>             "rescued": 0,
true =>             "skipped": 0,
true =>             "unreachable": 0
true =>         },
true =>         "master-02": {
true =>             "changed": 0,
true =>             "failures": 0,
true =>             "ignored": 0,
true =>             "ok": 1,
true =>             "rescued": 0,
true =>             "skipped": 0,
true =>             "unreachable": 0
true =>         },
true =>         "master-03": {
true =>             "changed": 0,
true =>             "failures": 0,
true =>             "ignored": 0,
true =>             "ok": 1,
true =>             "rescued": 0,
true =>             "skipped": 0,
true =>             "unreachable": 0
true =>         }
true =>     }
true => }
false => [WARNING]: Invalid characters were found in group names but not replaced, use
false => -vvvv to see details
```

Reply:
hi @clementblaise
I reproduced what you are describing, although the warning message is not the same, and I opened #27 

Thanks!

Reply:
hi @clementblaise!
I just start working on #28 which fix what you are facing. I you pull it you could check it. The issue is almost solved. I just want to cover the new code with some test.

I forced you same warning, creating and inventory group which contains a hyphen on its name and the warning message is not included on the json data
```shell
(fix_stderr)$ go run simple-ansibleplaybook-json.go
Go-ansible example ── [WARNING]: Invalid characters were found in group names but not replaced, use
Go-ansible example ── -vvvv to see details
[127.0.0.1] (Gathering Facts)
[127.0.0.1] (simple-ansibleplaybook)    Your are running 'simple-ansibleplaybook' example

Host: 127.0.0.1
 Changed: 0 Failures: 0 Ignored: 0 Ok: 2 Rescued: 0 Skipped: 0 Unreachable: 0
```

regards!

Reply:
Yes #28 needed be solved beforehand, thanks for the help @apenella !